lib/helpers/webview.ts

```diff
@@ -323,8 +324,8 @@ async function collectWebviewsDetails(
   for (const item of webviewsMapping) {
     detailCollectors.push(
       (async () => {
-        let port: number|undefined;
-        let host: string|undefined;
+        let port: number | undefined;
+        let host: string | undefined;
```

Comment:
by auto-format or something?

Reply:
Yes, by the pre-commit hook triggered via husky.